plugins/chat/db/migrate/20231214180001_update_relationship_between_chat_mentions_and_notifications.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class UpdateRelationshipBetweenChatMentionsAndNotifications < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+  BATCH_SIZE = 5000
+
+  def up
+    begin
+      inserted_count = DB.exec(<<~SQL, batch_size: BATCH_SIZE)
+        INSERT INTO chat_mention_notifications(chat_mention_id, notification_id)
+        SELECT cm.id, cm.notification_id
+        FROM chat_mentions cm
```

Comment:
After this migration has run, the existing instances can still write to `chat_mentions.notification_id` so we need to handle that in a post deploy migration as well.

Reply:
Oh, forgot about that, thank you!